client/src/commons/DatePick/DatePick.tsx

```diff
@@ -1,41 +1,48 @@
-import React from 'react';
-import { Variant } from '@material-ui/core/styles/createTypography';
-import { Typography, StandardTextFieldProps, TextField  } from '@material-ui/core';
-
-import { useStyles } from './DatePickStyles';
+import React from "react";
+import { Variant } from "@material-ui/core/styles/createTypography";
+import { StandardTextFieldProps } from "@material-ui/core";
+import DateFnsUtils from "@date-io/date-fns";
+import heLocale from "date-fns/locale/he";
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from "@material-ui/pickers";
+import { useStyles } from "./DatePickStyles";
+import { ParsableDate } from "@material-ui/pickers/constants/prop-types";
 
 const DatePick: React.FC<Props> = (props: Props): JSX.Element => {
-    const classes = useStyles({});
+  const classes = useStyles({});
 
-    const { lableText, lableTextVariant, type, ...rest } = props;
+  const { labelText, value, onChange } = props;
 
-    return (
-        <div className={classes.dateField}>
-            {
-                lableText && <Typography className={classes.dateText}>
-                    <b>{lableText + ':'}</b>
-                </Typography>
-            }
-            <div className={classes.dateText}>
-                <TextField
-                    InputLabelProps={{
-                        shrink: true
-                    }}
-                    id={type}
-                    type={type}
-                    className={classes.textField}
-                    size='small'
-                    label={lableText}
-                    {...rest}
-                />
-            </div>
-        </div>
-    );
+  return (
+    <MuiPickersUtilsProvider utils={DateFnsUtils} locale={heLocale}>
```

Comment:
The utils provider needs to sit at the root of the component tree, not each date picker